File: PYTHON/EntitasBioskop.py
class EntitasBioskop:
    def __init__(self, id_bioskop: str, nama: str, tahun_operasional: int):
        self._id = str(id_bioskop)
        self._nama = str(nama)
        self._tahun_operasional = int(tahun_operasional)

    # Getter & Setter
    def get_id(self) -> str:
        return self._id

    def set_id(self, id_bioskop: str):
        self._id = str(id_bioskop)

    def get_nama(self) -> str:
        return self._nama

    def set_nama(self, nama: str):
        self._nama = str(nama)

    def get_tahun_operasional(self) -> int:
        return self._tahun_operasional

    def set_tahun_operasional(self, tahun: int):
        self._tahun_operasional = int(tahun)

File: JAVA/EntitasBioskop.java
public class EntitasBioskop {
    protected String id;
    protected String nama;
    protected int tahunOperasional;

    public EntitasBioskop() {}

    public EntitasBioskop(String id, String nama, int tahunOperasional) {
        this.id = id;
        this.nama = nama;
        this.tahunOperasional = tahunOperasional;
    }

    // Getter & Setter
    public String getId() { return id; }
    public void setId(String id) { this.id = id; }

    public String getNama() { return nama; }
    public void setNama(String nama) { this.nama = nama; }

    public int getTahunOperasional() { return tahunOperasional; }
    public void setTahunOperasional(int tahunOperasional) { this.tahunOperasional = tahunOperasional; }
}